src/conformal_models/heuristics.jl

```diff
@@ -11,3 +11,23 @@ minus_softmax(y, ŷ) = 1.0 - ŷ
 Computes `abs(y - ŷ)` where `ŷ` is the predicted value.
 """
 absolute_error(y, ŷ) = abs(y - ŷ)
+
+@doc raw"""
+    ConformalBayes(y, fμ, fvar)
+computes the probability of observing a value y given a Gaussian distribution with mean fμ and a variance fvar.
+    inputs:
+        - y  the true values of the calibration set.
+        - fμ array of the mean values
+        - fvar array of the variance values
+
+    return:
+        -  the probability of observing a value y given a mean fμ and a variance fvar.
+"""
+function ConformalBayes(y, fμ, fvar)
```

Comment:
Can we use a different name for the function (or in any case not use camel case)